### PR TITLE
Refactor/symmetrical numbers

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In this pull request, i refactor symmetrical_numbers.py file in practice-1, update function main and add module symmetrical_numbers to main.py
#39

### CODE REVIEW

File: python/practices/practice-3/src/path1/symmetrical_numbers.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import os
+
+
+def reverse_digits(digits):
+    """
+    Reverse a list of digits
+
+    Parameters:
+        digits (list): A list of digits representing the number.
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/symmetrical_numbers.py]: I think `A list of digits` is good.

File: python/practices/practice-3/src/path1/symmetrical_numbers.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+"""
+Check and prints the numbers with the last 4 or last 5 digits or all 6 numbers
+that are symmetrical.
+"""
+
+
+import os
+
+
+def reverse_digits(digits):
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/symmetrical_numbers.py]: Please update the name and docstring, it doesn't match what this function does.

File: python/practices/practice-3/src/path1/symmetrical_numbers.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        digits (list): A list of digits representing the number.
+
+    Returns:
+        Returns a list that has been reversed compared to the original list
+    """
+
+    return digits == digits[::-1]
+
+
+def check_palindrome():
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/symmetrical_numbers.py]: Update the function's name.

File: python/practices/practice-3/src/path1/symmetrical_numbers.py (hunk 2 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path1/symmetrical_numbers.py]: Done

File: python/practices/practice-3/src/path1/symmetrical_numbers.py (hunk 3 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path1/symmetrical_numbers.py]: Done

**Pull request merged**